tests/unit_tests/test_wave_06.py

```diff
@@ -75,8 +75,16 @@ def test_swap_best_by_category():
         my_priority="Clothing",
         their_priority="Decor"
     )
-
-    raise Exception("Complete this test according to comments below.")
+    assert result is True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_c not in tai.inventory
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
+    assert item_f not in jesse.inventory
```

Comment:
Love that you used a more flexible, `item in person.inventory` syntax here over a stricter `list_of_items[index] == item`! Unit testing is truly an art over a science, and having flexible and accurate tests do make a big difference down the road!

Reply:
To make this test even more accurate, we can also assert:
```
assert item_f in tai.inventory
assert item_c in jesse.inventory
```